internal/service/elbv2/target_group_test.go

```diff
@@ -4836,7 +4936,7 @@ resource "aws_vpc" "test" {
 `, rName, protocol)
 }
 
-func testAccTargetGroupConfig_stickinessValidity(rName, protocol, stickyType string, enabled bool) string {
+func testAccTargetGroupConfig_stickinessValidity(rName, protocol, stickyType string, enabled bool, loadBalanceAlgorithmType string) string {
```

Comment:
Added to assist in checking for invalid stickiness configuration.  `weighted_random` does not support stickiness.